stdlib/email/message.pyi

```diff
@@ -72,22 +72,22 @@ class Message:
     # Morally, the return type should be `AnyOf[_HeaderType, None]`,
     # which we could spell as `_HeaderType | Any`,
     # *but* `_HeaderType` itself is currently an alias to `Any`...
```

Comment:
(see other comment, github wouldn't let me do a suggestion across a deleted line)
```suggestion
```
```suggestion
    # so using "the Any trick" instead.
```
On that note, we never ended up making an alias for it, something like `MaybeNone`. https://github.com/python/typeshed/issues/11094